ush/forecast_postdet.sh

```diff
@@ -524,9 +524,15 @@ CICE_postdet() {
     local seconds
     seconds=$(to_seconds "${restart_date:8:2}0000")  # convert HHMMSS to seconds
     cice_restart_file="${DATArestart}/CICE_RESTART/cice_model.res.${restart_date:0:4}-${restart_date:4:2}-${restart_date:6:2}-${seconds}.nc"
+    if [[ "${DO_JEDIOCNVAR:-NO}" = "YES" && "${restart_date}" = "${sdate_current_cycle}" ]]; then
```

Comment:
@guillaumevernieres Two things, one i changed the sdate_current_cycle name on you to be: model_start_date_current_cycle  -- sorry! 

And 2, I don't think you can have RERUN=YES if restart_date=model_start_date_current_cycle, so I don't think this is a  needed block.... but we should get someone else to confirm.

Reply:
arf ... yes, you also told me about it. I'll remove the block too.

Reply:
did you push that change @JessicaMeixner-NOAA ? I don't see `model_start_date_current_cycle` defined anywhere.

Reply:
just pushed.. sorry i thought i had